src/mergeSort.js

```diff
@@ -0,0 +1,33 @@
+//time complexity: O(nlogn)
+
+export default function mergeSort(arr) {
+  const arrLength = arr.length
+  if(arrLength < 2){
+    return arr
+  }
+
+  let middle = parseInt(arrLength/2)
+  var left = arr.slice(0, middle)
```

Comment:
```var``` ☹️ 

My only issue with ```var``` is that you should be consistent. If you are using ES6 syntax like ```let``` then it makes more sense to use ```const``` instead of ```var``` IMHO